### PR TITLE
#167482222 Change create article interface and add description field

### PR DESCRIPTION
**Pull request by Niyitangasam**
Description
=======
As Po requested, this PR adds.

- Description when creating a new article
- Add feature image on an article
- Refactor CSS to make create article interface and viewing article looking good.

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[167482222](https://www.pivotaltracker.com/n/projects/2326984/stories/167482222)

Screenshots
=======

image

### CODE REVIEW

File: src/assets/scss/_oneStory.scss [hunk 1]
```diff
@@ -36,18 +36,28 @@
...
   cursor: pointer;
+  position: relative;
   top: -10px;
   .ratings {
+    width: 15px;
+    height: 34px;
     margin: 0 5px;
     opacity: 0.7;
-    height: 16px !important;
-    width: 61% !important;
```
Comment (AJAkimana) [inline src/assets/scss/_oneStory.scss]: These CSSes are used by rating, I suggest not to delete them

**Review by AJAkimana — COMMENTED** [review]
Good work @Niyitangasam, I left inline comments about the styles you removed. Look about it and resolve them.

File: src/assets/scss/_oneStory.scss (hunk 1 above)
Comment (ericnyirimana) [reply src/assets/scss/_oneStory.scss]: @Niyitangasam yes I agree with @AJAkimana

**Review by ericnyirimana — APPROVED** [review]

**Review by AJAkimana — APPROVED** [review]
Looks great

**Pull request merged**